level_five.gd

```diff
@@ -66,6 +70,11 @@ func handle_spawns():
 		emit_signal("spawn_wave", new_arr)
 
 
+func handle_boons():
+	if Game.score > 100:
+		emit_signal("spawn_boons")
```

Comment:
This will emit this signal every frame forever after your score hits 100

Reply:
Yep, I saw this happening at the end of the night. Going to figure out how to fix it today